### PR TITLE
Add Python 3.8 support and remove Python 3.4 support

### PR DESCRIPTION
**Pull request by eduardo-rodrigues**

### CODE REVIEW

**Comment by chrisburr** [comment]
Can you add a `MANIFEST.in` while you're at it to include the LICENCE file and tests in the source tarball on PyPI?

File: setup.py [hunk 1]
```diff
@@ -32,6 +35,7 @@ def get_version():
     license = 'new BSD',
```
Comment (chrisburr) [inline setup.py]:
```suggestion
    license = 'BSD-3-Clause',
```

**Comment by eduardo-rodrigues** [comment]
I can for sure; just need to remind myself of the syntax as it's been long since I prepared one.

Do you really find it useful to have the tests on PyPI?

**Comment by chrisburr** [comment]
> I can for sure; just need to remind myself of the syntax as it's been long since I prepared one.

I think you need:

```
include README.rst LICENSE
recursive-include tests
```

> Do you really find it useful to have the tests on PyPI?

It makes the tarball slightly more useful to package managers as they can full test any packages they build without needing to access the original repository. It's a niche use case but as they're very small I see no reason not to include them.

**Comment by eduardo-rodrigues** [comment]
Thanks - was about to commit the addition of the README to the MANIFEST.in file but you were faster :+1:.

**Pull request merged**